src/core/ColorPicker/ColorInputPanel.tsx

```diff
@@ -250,6 +284,38 @@ export const ColorInputPanel = React.forwardRef(
             }}
           />
         </InputContainer>
+        {showAlpha && (
+          <InputContainer
+            status={
+              Number(input[3]) < 0 || Number(input[3]) > 1
+                ? 'negative'
+                : undefined
+            }
+          >
+            <Input
+              size='small'
+              type='number'
+              min='0'
+              max='1'
+              step='.01'
```

Comment:
Need to add step to the other inputs too, otherwise you get this message when hovering over the input
![image](https://user-images.githubusercontent.com/88331924/141128391-9969b77c-50cc-429f-a415-ca1aceab917e.png)

Reply:
done